docs/examples/manifests/mssql-backup.yaml

```diff
@@ -6,5 +6,5 @@ metadata:
 spec:
   includedNamespaces:
   - mssql-persistent
-  storageLocation: dpa-sample-1
+  storageLocation: d-velero-sample-1
```

Comment:
looks good so far :) qq.  why change the storage location name, is that temporary?

Reply:
I will remove this, this was just for testing!